packages/razzle/scripts/build.js

```diff
@@ -0,0 +1,132 @@
+#! /usr/bin/env node
+// Do this as the first thing so that any code reading it knows the right env.
+process.env.NODE_ENV = 'production';
+
+// Makes the script crash on unhandled rejections instead of silently
+// ignoring them. In the future, promise rejections that are not handled will
+// terminate the Node.js process with a non-zero exit code.
+process.on('unhandledRejection', err => {
+  throw err;
+});
+
+require('dotenv').config({ silent: true });
+const webpack = require('webpack');
+const fs = require('fs-extra');
+const chalk = require('chalk');
+const path = require('path');
+const paths = require('../config/paths');
+const printErrors = require('../config/printErrors');
+const createConfig = require('../config/create-config');
+const FileSizeReporter = require('react-dev-utils/FileSizeReporter');
+const measureFileSizesBeforeBuild = FileSizeReporter.measureFileSizesBeforeBuild;
+const printFileSizesAfterBuild = FileSizeReporter.printFileSizesAfterBuild;
+
+// First, read the current file sizes in build directory.
+// This lets us display how much they changed later.
+measureFileSizesBeforeBuild(paths.appBuildPublic).then(previousFileSizes => {
+  // Remove all content but keep the directory so that
+  // if you're in it, you don't end up in Trash
+  fs.emptyDirSync(paths.appBuild);
+
+  // Start the webpack build
+  build(previousFileSizes);
+
+  // Merge with the public folder
+  copyPublicFolder();
+});
+
+function build(previousFileSizes) {
+  // Check if razzle.config.js exists
+  let razzle = {};
+  try {
+    razzle = require(paths.appRazzleConfig);
+  } catch (e) {}
```

Comment:
Is there any reason you are swallowing the errors here? Sometime the razzle.config.js exists and isn't being picked and would be nice to log an info message and the stack.